nvtabular/loader/backend.py

```diff
@@ -34,21 +36,20 @@ def _augment_schema(
     for label in conts or []:
         schema[label] = schema[label].with_tags(Tags.CONTINUOUS)
 
-    # Set the appropriate properties for the sparse_names/sparse_max/sparse_as_dense
-    for col in sparse_names or []:
+    for col in padded_cols or []:
         cs = schema[col]
-        properties = cs.properties
-        if sparse_max and col in sparse_max:
-            properties["value_count"] = {"max": sparse_max[col]}
-        if sparse_as_dense:
-            properties["value_count"]["min"] = properties["value_count"]["max"]
+        dims = Shape(((1, batch_size), None))
```

Comment:
What is the `batch_size` required for?

Reply:
Not required for anything specific, just following the principle that the schema should always accurately reflect the data to the greatest extent possible. Here we have shape information since we know the batch size, so we fill it in in case that helps something downstream. I don't know if it actually will, but it seemed like the right thing to do.